### PR TITLE
green output

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
This PR has two changes to the `codeclimate init` command config generation:

*minor*
1) make successful output messaging green

*major*
2) generate default configs if possible, even if `.codeclimate.yml` already exists. Keep same error / exit code 1 behavior. 

Since it might still be useful for users with a `.codeclimate.yml` to get default configs by running
`codeclimate init`, this change generates applicable configs even when
.codeclimate.yml present, before finishing with the same warning message and
exit code 1:

```
$ codeclimate init
Config file .csslintrc successfully generated.
Skipping generating coffeelint.json file (already exists).
Config file .eslintrc successfully generated.
Skipping generating .rubocop.yml file (already exists).
Config file .codeclimate.yml already present.
Try running 'validate-config' to check configuration.
```

cc @codeclimate/review

### CODE REVIEW

File: lib/cc/cli/init.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class Init < Command
 
       def run
         if !upgrade? && filesystem.exist?(CODECLIMATE_YAML)
+          create_default_configs
           fatal "Config file .codeclimate.yml already present.\nTry running 'validate-config' to check configuration."
```
Comment (GordonDiggs) [inline lib/cc/cli/init.rb]: It seems odd to have a fatal output if we successfully created default configs. Maybe the message should mention that we generated them?

Comment (GordonDiggs) [inline lib/cc/cli/init.rb]: EDIT: nvm - I see the output of that now

**Comment by GordonDiggs** [comment]
LGTM

File: lib/cc/cli/init.rb (hunk 1 above)
Comment (ABaldwinHunter) [inline lib/cc/cli/init.rb]: Yeah, I had qualms about the final exitcode as well.

Comment (wfleming) [inline lib/cc/cli/init.rb]: Yeah, if this default configs is the desired behavior, I think it should be a successful output exit code & message. I would suggest outputting the informational message about not changing `.codeclimate.yml` first, and then generate the default configs.

This can & probably should actually be refactored to be part of `#create_codeclimate_yaml` behavior. Putting it here make behavior kind of strange in that `codeclimate init` will not touch `.codeclimate.yml`, but will touch default engine configs, whereas `codeclimate init --upgrade` will refuse to generate default engine configs if `.codeclimate.yml` already has engines.

Comment (ABaldwinHunter) [inline lib/cc/cli/init.rb]: @wfleming cool. Thanks for the explanation. I'd like consistently generating a successful exit code.

What was our original reason in favor of going with the `exitcode 1` when there's no `.codeclimate.yml`? What would you think about something more neutral like `.codeclimate.yml already present, skipping generation`, like we do for each engine config that's already present?

Comment (wfleming) [inline lib/cc/cli/init.rb]: > What was our original reason in favor of going with the exitcode 1 when there's no .codeclimate.yml

The logic predated me, but it was failing with exit code 1 if `.codeclimate.yml` **already existed** because `init`'s primary purpose was to create that file.

> What would you think about something more neutral like .codeclimate.yml already present, skipping generation

I think that sounds fine (`#warn` seems appropriate for the output). The verb "generation" would not be appropriate for the `--upgrade` case, though: it seems like you'll need another `*_verb` method on the generator class to distinguish between the generated/upgraded.

**Comment by ABaldwinHunter** [comment]
@codeclimate/review @wfleming Ready for re-review.

I updated the PR to return a warning that `.codeclimate.yml` is not found, instead of error.

I left the behavior of [init --upgrade](https://github.com/codeclimate/codeclimate/pull/210/files#diff-ebd3ae72c4e4074aa627b08e8ed743bbL13) the same because it seems separate enough, and perhaps desirable as is: at the moment, `codeclimate init` doesn't get called on .com unless the `.codeclimate.yml` file is missing or contains engines, so this change shouldn't have an impact there.

Let me know what you think!

File: lib/cc/cli/help.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@ def run
 
       def commands
         [
-          "analyze [-f format]",
+          "analyze [-f format] [-e engine] <path>",
```
Comment (GordonDiggs) [inline lib/cc/cli/help.rb]: I don't think it needs to be removed, but want to mention that this seems like a drive-by fix

File: lib/cc/cli/command.rb [hunk 3]
```diff
@@ -24,6 +24,10 @@ def execute
         run
       end
 
+      def congrats(message)
```
Comment (GordonDiggs) [inline lib/cc/cli/command.rb]: WDYT of `success`?

Comment (ABaldwinHunter) [inline lib/cc/cli/command.rb]: k. I like that

File: lib/cc/cli/help.rb (hunk 2 above)
Comment (ABaldwinHunter) [inline lib/cc/cli/help.rb]: yeah, agreed.

At first I was curious about whether we have `--upgrade` in the help commands, and noticed this absence as well.

**Comment by ABaldwinHunter** [comment]
@GordonDiggs @wfleming fixed!

**Comment by GordonDiggs** [comment]
LGTM
